apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java

```diff
@@ -187,6 +231,11 @@ private static Arguments newFirstClaimInvalidLineNumberCase() {
         expectedException);
   }
 
+  /**
+   * Recurring claim test case function.
```

Comment:
Since these are parameterized tests, we don't have a real spot to explain each test case; this would be the spot. What are we actually testing here? What is a test success? What about the recurring claim is being tested? These are the kinds of things I'd want to know looking at this as a maintainer; the intention of the test.

Reply:
done